tests/php/Core/Infrastructure/Security/ProviderConfiguration/OpenId/Api/UpdateOpenIdConfiguration/UpdateOpenIdConfigurationControllerTest.php

```diff
@@ -40,128 +40,131 @@
-        $this->container->expects($this->any())
-            ->method('has')
-            ->willReturn(true);
-        $this->container->expects($this->any())
-            ->method('get')
-            ->withConsecutive(
-                [$this->equalTo('security.authorization_checker')],
-                [$this->equalTo('parameter_bag')]
-            )
-            ->willReturnOnConsecutiveCalls(
-                $authorizationChecker,
-                new class () {
-                    public function get(): string
-                    {
-                        return __DIR__ . '/../../../../../';
-                    }
-                }
-            );
-
-        $this->request = $this->createMock(Request::class);
-    }
-
-    /**
-     * Test that a correct exception is thrown when body is invalid.
-     */
-    public function testCreateUpdateOpenIdConfigurationRequestWithInvalidBody(): void
-    {
-        $controller = new UpdateOpenIdConfigurationController();
-        $controller->setContainer($this->container);
-
-        $invalidPayload = json_encode([]);
-        $this->request
-            ->expects($this->once())
-            ->method('getContent')
-            ->willReturn($invalidPayload);
-
-        $this->expectException(\InvalidArgumentException::class);
-        $controller($this->useCase, $this->request, $this->presenter);
-    }
-
-    /**
-     * Test that the controller correctly send the request to the useCase with valid body.
-     */
-    public function testCreateUpdateOpenIdConfigurationRequestWithValidBody(): void
-    {
-        $controller = new UpdateOpenIdConfigurationController();
-        $controller->setContainer($this->container);
-
-        $validPayload = json_encode([
-            'is_active' => true,
-            'is_forced' => true,
-            'trusted_client_addresses' => [],
-            'blacklist_client_addresses' => [],
-            'base_url' => 'http://127.0.0.1/auth/openid-connect',
-            'authorization_endpoint' => '/authorization',
-            'token_endpoint' => '/token',
-            'introspection_token_endpoint' => '/introspect',
-            'userinfo_endpoint' => '/userinfo',
-            'endsession_endpoint' => '/logout',
-            'connection_scopes' => [],
-            'login_claim' => 'preferred_username',
-            'client_id' => 'MyCl1ientId',
-            'client_secret' => 'MyCl1ientSuperSecr3tKey',
-            'authentication_type' => 'client_secret_post',
-            'verify_peer' => false
-        ]);
-
-        $this->request
-            ->expects($this->exactly(2))
-            ->method('getContent')
-            ->willReturn($validPayload);
-
-        $this->useCase
-            ->expects($this->once())
-            ->method('__invoke');
-
-        $controller($this->useCase, $this->request, $this->presenter);
-    }
+    //@todo: Reimplement those tests while handling update openid configuration extension.
```

Comment:
use `->skip('Reimplement those tests while handling update openid configuration extension');` instead of comment tests